src/Symfony/Bridge/Twig/Extension/HttpFoundationExtension.php

```diff
@@ -60,6 +60,10 @@ public function generateAbsoluteUrl($path)
             return $path;
         }
 
+        if ('/' === $path[0]) {
```

Comment:
`$path` can be empty, you need to test that. So, that can even be merged with the condition below.